backup_and_restore/index.adoc

```diff
@@ -27,3 +26,32 @@ You might run into several situations where {product-title}  does not work as ex
 * You have lost the majority of your control plane hosts, leading to etcd quorum loss.
 
 You can always recover from a disaster situation by xref:../backup_and_restore/control_plane_backup_and_restore/disaster_recovery/scenario-2-restoring-cluster-state.adoc#dr-restoring-cluster-state[restoring your cluster to its previous state] using the saved etcd snapshots.
+
+[id="application-backup-restore-operations-overview"]
+== Application backup and restore operations
+
+As a cluster administrator, you can back up and restore applications running on {product-title} by using the OpenShift API for Data Protection (OADP).
+
+OADP backs up and restores Kubernetes resources, internal images, and cluster metadata, at the granularity of a namespace, by using a supported version of link:https://velero.io/[Velero]. See xref:../backup_and_restore/application_backup_and_restore/oadp-features-plugins.adoc#oadp-features_oadp-features-plugins[OADP features].
```

Comment:
I think that we may want to consider removing cluster metadata, as I don't know which metadata specifically oadp/velero will backup.